### PR TITLE
Push Changes from Master to Prod Sync 

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Push changes from master to prod sync.
> Fixes #

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: community-new.html [hunk 1]
```diff
@@ -600,6 +600,21 @@ <h5 style="margin-bottom: 18px;">Ballerina: A Language to Develop, Consume and C
                 </td>	
                 <td class="cEventURL"><a class="cEventRegistration" href="https://www.meetup.com/sfjava/events/284669431" target="_blank" style="text-align: center;">More info</a></td>	
             </tr>
+            <tr class="event-expiry"  data-expiry="March 24, 2022 19:00:00">
```
Comment (anupama-pathirage) [inline community-new.html:603]: What   is this expiry date? Is  it  correct?

File: community/events.html [hunk 2]
```diff
@@ -371,11 +371,22 @@ <h5 style="margin-bottom: 18px;">Ballerina: A Language to Develop, Consume and C
                     </td>	
                     <td class="cEventURL"><a class="cEventRegistration" href="https://www.meetup.com/sfjava/events/284669431" target="_blank" style="text-align: center;">More info</a></td>	
                 </tr>
-                
-                
-                
-                
-                
+                 <tr class="event-expiry"  data-expiry="March 24, 2022 19:00:00">
```
Comment (anupama-pathirage) [inline community/events.html:374]: What is this  expiry date?

File: community-new.html (hunk 1 above)
Comment (praneesha) [reply community-new.html:603]: @ashweni

Comment (ashweni) [reply community-new.html:603]: @dinushan20

Comment (dinushan20) [reply community-new.html:603]: +1

Comment (dinushan20) [reply community-new.html:603]: Actually  that code was used to automatically unpublished the event after it. At the moment it is not working, For now we can remove this

**Comment by dinushan20** [comment]
+1

File: community/events.html (hunk 2 above)
Comment (dinushan20) [reply community/events.html:374]: +1

**Review by keizer619 — APPROVED** [review]

**Pull request merged**